src/components/App.css

```diff
@@ -1,4 +1,5 @@
-.map-container {
+.map-container,
+.leaflet-container {
```

Comment:
let's set `.leaflet-container` to use a separate set of styles `width: 100%` and `height: 100%` instead of using the same positioning from `.map-container`. This allows it to by styled without needing to have a tight coupling to its parent container.

Related to that, let's put that style in `src/components/Map/Map.css`.